wire-ios-sync-engine/Tests/Source/SessionManager/APIMigrationManagerTests.swift

```diff
@@ -248,22 +249,40 @@ class APIMigrationManagerTests: MessagingTest {
             pushChannel: MockPushChannel()
         )
 
-        return ZMUserSession(
-            userId: .create(),
-            transportSession: mockTransportSession,
-            mediaManager: MockMediaManager(),
-            flowManager: FlowManagerMock(),
+        let mockContextStorable = MockLAContextStorable()
+        mockContextStorable.clear_MockMethod = { }
+        let configuration = ZMUserSession.Configuration()
+
+        var builder = ZMUserSessionBuilder()
+        builder.withAllDependencies(
```

Comment:
Why not setting the properties directly?

```suggestion
        builder.appVersion = "999"
        builder.application = application
        ...
```

Reply:
Because we derive new objects from all the passed ones and all are needed. They have some cross-dependencies too.